nodeJS/express/views.md

```diff
@@ -147,48 +161,37 @@ You can insert this component into another EJS file like so:
 </html>
 ```
 
-This can be used to include headers and footers in all of your pages, for example.
-
-Note that the navbar expects a `links` value. To pass this data into the navbar, you can pass it when rendering `index.ejs` which contains the navbar. Modify `app.js` such that a `links` object is defined and passed into the `render` function in the `"/"` route handler:
-
-```javascript
-// app.js
-const links = [
-  { href: "/", text: "Home" },
-  { href: "about", text: "About" },
-];
+This will include the navbar with links in `index.ejs`. Reusable templates can be used to include headers and footers in all of your pages.
```

Comment:
Nitty suggestion
```suggestion
This will include the navbar with links in `index.ejs`. Run your server and see! Reusable templates can be used to include headers and footers in all of your pages.
```

Reply:
Have added!